### ISSUE TITLE
Cannot add reddit extension

### ISSUE THREAD

**Issue by ghost** [Ranchero-Software/NetNewsWire#3479]
I get the following error in NetNewsWire after entering my user ID and PW on the reddit oAuth page

> The operation couldn’t be completed. (OAuthSwiftError error -10.)

**Comment by vincode-io** [comment]
Is this on iOS or macOS? Were you having any network problems at the time this happened?

**Comment by LeeTeng2001** [comment]
me too, it's on macos

**Comment by brentsimmons** [comment]
Is it working now?

**Comment by LeeTeng2001** [comment]
![Screenshot 2023-01-09 at 2 14 32 PM](https://user-images.githubusercontent.com/63984709/211249940-b3207433-ca39-452b-8560-84271dfed074.jpg)
Nope

**Comment by LeeTeng2001** [comment]
But I did received a confirmation email from reddit that netnewswire has already authorized my reddit account

**Comment by vincode-io** [comment]
@LeeTeng2001 have you tried rebooting your Mac and trying again? If that didn't work, could you let me know which version of macOS you are using?

**Comment by LeeTeng2001** [comment]
Hi, it's working now, but I'm unsure why. I've recently moved from China to Malaysia, I'm using a VPN to access reddit in China, thank you for your help

**Comment by mantia** [comment]
Still get the same situation.

The operation couldn’t be completed. (OAuthSwiftError error -5.)

**Comment by brentsimmons** [comment]
I just saw the same thing — which is good, because it makes it much easier to fix when I can reproduce the bug.

**Comment by MacawsArcher** [comment]
This is happening to me on iOS (16.3 and 16.4).

**Comment by MacawsArcher** [comment]
> This is happening to me on iOS (16.3 and 16.4).

Just was able to add my account

**Comment by ashishgandhi** [comment]
With the Reddit API changes, this can be closed?

**Comment by MacawsArcher** [comment]
Yes it can be closed.On Jun 14, 2023, at 03:37, Ashish Gandhi ***@***.***> wrote:﻿
With the Reddit API changes, this can be closed?
